datadog-service/0.15.0/README.md

```diff
@@ -0,0 +1,200 @@
+# datadog-service
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/keptn-sandbox/datadog-service)
+[![Go Report Card](https://goreportcard.com/badge/github.com/keptn-sandbox/datadog-service)](https://goreportcard.com/report/github.com/keptn-sandbox/datadog-service)
+
+This implements the `datadog-service` that integrates the [Datadog](https://en.wikipedia.org/wiki/Datadog) observability platform with Keptn. This enables you to use Datadog as the source for the Service Level Indicators ([SLIs](https://keptn.sh/docs/0.15.x/reference/files/sli/)) that are used for Keptn [Quality Gates](https://keptn.sh/docs/concepts/quality_gates/).
+If you want to learn more about Keptn visit us on [keptn.sh](https://keptn.sh)
+
+Check the issue on the main repo for more info: https://github.com/keptn/keptn/issues/2652
+
+
+## Quickstart
+If you are on Mac or Linux, you can use [examples/kup.sh](https://raw.githubusercontent.com/keptn-sandbox/datadog-service/release-0.15.0/examples/kup.sh) to set up a local Keptn installation that uses Datadog. This script creates a local minikube cluster, installs Keptn, Istio, Datadog and the Datadog integration for Keptn (check the script for pre-requisites). 
+
+To use the script,
+```bash
+$ export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com" 
+$ examples/kup.sh
```

Comment:
General comment which is valid for all your `bash`  examples: remove the `$ ` in front.
while it might look more like a command line, it's very annoying if you want to copy the commands from the readme into your terminal.

```suggestion
export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com" 
examples/kup.sh
```

Reply:
Fixed. I did think about removing the `$` but thought otherwise (since it allows you to visually separate one command from the other)